### PR TITLE
added to GCI

### PR DESCRIPTION
**Pull request by pollend**
just added myself to GCI

### CODE REVIEW

**Comment by Cervator** [comment]
@smsunarto could we add another column for key words that the mentor can help with, or would that not fit? Can we tweak time zone to "Time zone / general availability" or something? So I might be "US East - afternoons and evenings most days)

Or rather than list time zone and generic comments like that we could all list our generally available hours in UTC? For consistency and brevity

File: _mentors/pollend.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: Michadl Pollind
```
Comment (aaron-harris) [inline _mentors/pollend.md]: I think you have a typo here...

**Comment by smsunarto** [comment]
@Cervator if you check the actual page, the timezone gets automatically converted to the mentor current local time. So it's not generic, it's the convention that the javascript can understand and translate to actual time. I probably can add a "more information" button or link that will create some sort of pop up = more space.

**Comment by Cervator** [comment]
Ooooh - neat!

File: _mentors/pollend.md (hunk 1 above)
Comment (pollend) [reply _mentors/pollend.md]: oh no

**Comment by smsunarto** [comment]
Hi @pollend, I recently update the mentor file format. Please update it :D

**Pull request merged**